vme/src/unit_data.h

```diff
@@ -11,36 +11,90 @@
 class unit_data : public basedestruct
 {
 public:
+    static constexpr sbit16 MinAlignment = -1000; ///< Minimum possible value for alignment
+    static constexpr sbit16 MaxAlignment = 1000;  ///< Maximum possible value for alignment
+
+public:
+    /**
+     * @name Constructors/Destructor
+     * @{
+     */
+
+    /**
+     * @param unit_type One of UNIT_ST_NPC, UNIT_ST_PC, UNIT_ST_ROOM, UNIT_ST_OBJ
+     */
     explicit unit_data(ubit8 unit_type);
-    virtual ~unit_data();
+    unit_data(const unit_data &) = delete;
```

Comment:
Haaa... My eyes are burning :-D  That's pretty absurd. What does it mean :-) It is really necessary?